### PR TITLE
[Merged by Bors] - Update dependabot.yml

### PR DESCRIPTION
**Pull request by morenol**
but also enabling minor updates

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -35,6 +35,5 @@ updates:
     ignore:
       - dependency-name: "*"
         update-types: [
-            "version-update:semver-minor",
             "version-update:semver-patch",
           ] # Only major version updates
```
Comment (morenol) [inline .github/dependabot.yml:39]:
```suggestion
          ]
```

**Review by tjtelan — APPROVED** [review]
bors r+

**Comment by morenol** [comment]
bors retry

**Comment by morenol** [comment]
bors retry

**Pull request closed without merging**